zerver/actions/create_user.py

```diff
@@ -172,6 +177,12 @@ def set_up_streams_for_new_human_user(
         acting_user=acting_user,
     )
 
+    bulk_add_members_to_user_groups(
+        user_groups,
+        [user_profile.id],
+        acting_user=acting_user,
+    )
```

Comment:
This should probably check if `user_groups` is nonempty, or maybe we could move that into the `bulk_add_members_to_user_groups` function, since I think at least one existing caller already wraps it with such a check. Can be a follow-up commit.

Would save a query for the common case that there are no groups to add.